encoding.bs

```diff
@@ -1234,14 +1233,13 @@ constructor, when invoked, must run these steps:
 method, when invoked, must run these steps:
 
 <ol>
- <li><p>If the <a for=TextDecoder>do not flush flag</a> is unset, set <a for=TextDecoder>decoder</a>
- to a new <a for=TextDecoderCommon>encoding</a>'s <a for=/>decoder</a>, set
- <a for=TextDecoder>stream</a> to a new <a for=/>stream</a>, and unset the
- <a for=TextDecoderCommon>BOM seen flag</a>.
+ <li><p>If <a for=TextDecoder>do not flush</a> is false, set <a for=TextDecoder>decoder</a>
```

Comment:
Another problem with this text is that it doesn't say where to grab "do not flush" from. See #175 for an idea of how to address that. Although I guess I'd accept it if you don't want to take that on as well.

Reply:
Both this change and updating cases of "this object" to "this" are changes that go a bit outside the scope of this pull request. Since #175 seems to be dead, maybe some similar PR, though smaller in scope, should be opened.